### PR TITLE
fpgainfo: add --force option

### PR DESCRIPTION
**Pull request by luebbers**
Specifying --force will try clearing errors up to 64
times. This may be necessary on certain error
conditions to flush in-flight transactions.

### CODE REVIEW

File: tools/base/fpgainfo/fpgaerr.py [hunk 1]
```diff
@@ -285,6 +286,13 @@ def args(self, parser):
                             default='fme',
                             help='specify what kind of errors to operate on')
 
+        # FIXME: should get a '-f' option later
```
Comment (shivarao) [inline tools/base/fpgainfo/fpgaerr.py]: Seems like a easy FIXME to knock out now

**Comment by shivarao** [comment]
This seems ok except that we'll need to document the --force flag now.

What's the risk of making the --force behavior the default behavior? It should result in one clear on systems that don't require multiple clears.

File: tools/base/fpgainfo/fpgaerr.py (hunk 1 above)
Comment (luebbers) [reply tools/base/fpgainfo/fpgaerr.py]: "-f" is currently used for the PCI function; as part of consolidating all tools, that should be turned into "-F", but that would change the current usage; we'll need to coordinate that carefully.

**Comment by luebbers** [comment]
Having to clear the same error multiple times does not seem to be a feature to me - I'd rather go back and reexamine the clearing flow so that it's not necessary.

**Comment by luebbers** [comment]
If it's any help, the `--force` flag should really rarely be necessary, since we're catching the error condition early now.

I think having the default behavior not clear rare (and avoidable) error conditions helps guide the users towards understanding what's causing those conditions (i.e. uncoordinated PR). If we blanket-hide the symptoms, we end up with a less clean solution.

**Pull request merged**